paper-dialog-common.css

```diff
@@ -9,15 +9,15 @@ subject to an additional IP rights grant found at http://polymer.github.io/PATEN
 */
 
 :host {
-  display: block;
-  margin: 24px 40px;
-  -webkit-overflow-scrolling: touch;
+  @apply(--paper-font-body1);
+  @apply(--shadow-elevation-16dp);
 
+  -webkit-overflow-scrolling: touch;
   background: var(--paper-dialog-background-color, --primary-background-color);
   color: var(--paper-dialog-color, --primary-text-color);
+  display: block;
+  margin: 24px 40px;
 
-  @apply(--paper-font-body1);
```

Comment:
Like the other Polymer elements, if we can (because is cascading), set the external `apply()`s at the top. I turn back?